crypto/ake/enclave/src/lib.rs

```diff
@@ -41,16 +41,20 @@ const MAX_AUTH_PENDING_REQUESTS: usize = 64;
 const MAX_PEER_SESSIONS: usize = 64;
 
 /// Maximum number of concurrent sessions to this enclave from router enclaves.
-const MAX_FRONTEND_SESSIONS: usize = 500;
+const MAX_FRONTEND_SESSIONS: usize = 64;
```

Comment:
This should be a many to one where there are Many backends to one frontend, as such each shard should only be talking to a handful of routers. I used 64 since that was the same as the peer sessions.
Each one of these connections is 2096. So this reduces ~1MB to 128KB